common/source/docs/common-blheli32-passthru.rst

```diff
@@ -1,62 +1,149 @@
+.. warning::
+   For pass-through to function, the autopilot must be configured to use one of the DShot protocols.  If you wish to eventually use one of the other protocols (e.g. PWM, OneShot125) that the ESC supports, you may still configure the ESCs using Pass-Through (e.g. change motor directions, set min/max values, etc) but then finally re-configure the autopilot to *not* use DShot.  Once the autopilot and ESCs are rebooted the ESC should auto-detect that the ESCs are no longer using DShot.
 
-Now connect a USB cable to your autopilot and use BLHeliSuite32 on Windows to connect. Select "BLHeli32 Bootloader (Betaflight/Cleanflight)" from the interfaces menu.
+..  youtube:: np7xXY_e5sA
+    :width: 100%
 
-.. image:: ../../../images/blhelisuite32.jpg
-    :target: ../_images/blhelisuite32.jpg
+.. _blheli32-reversible-dshot:
 
 Reversible DShot ESCs
 ---------------------
 
-Currently, only BLHeli32 and BLHeli-S capable reversible DShot ESCs are supported. In order to use one, the output which drives it must be designated with the appropriate bit in the :ref:`SERVO_BLH_3DMASK<SERVO_BLH_3DMASK>` bitmask parameter. This will map the outputs 1000-1500-2000 values to the correct digital values for the ESC to provide FullReverse-Idle-FullForward range operation, respectively.
+Currently, only BLHeli32 and BLHeli_S capable reversible DShot ESCs are supported. In order to use one, the output which drives it must be designated with the appropriate bit in the :ref:`SERVO_BLH_3DMASK<SERVO_BLH_3DMASK>` bitmask parameter. This will map the outputs 1000-1500-2000 values to the correct digital values for the ESC to provide FullReverse-Idle-FullForward range operation, respectively.
 
-If the craft has been setup for DShot commands then ArduPilot will supply the correct command at startup in order to set the ESCs in reversible mode.
+If :ref:`DShot commands <dshot-commands>` have been enabled then ArduPilot will automatically configure the ESCs to reversible mode at startup.  If DShot commands have not been configured you must manually configure the ESCs' "Motor Direction" to "Bidirectional 3D" as shown below.
 
-In a similar fashion, normal output rotation direction can be reversed by setting :ref:`SERVO_BLH_RVMASK<SERVO_BLH_RVMASK>` without any changes needing to be made through ESC setup software (e.g. BLHeliSuite). This can also be used on ESCs with forward and reversed active operation, ie reversible ESCs, to set the "forward" direction's rotation.
+  .. image:: ../../../images/blheli-reversible-dshot.png
+    :target: ../_images/blheli-reversible-dshot.png
+
+In a similar fashion, normal output rotation direction can be reversed by setting :ref:`SERVO_BLH_RVMASK<SERVO_BLH_RVMASK>` without any changes needing to be made through ESC configuration software (e.g. BLHeliSuite). This can also be used on ESCs with forward and reversed active operation, ie reversible ESCs, to set the "forward" direction's rotation.
```

Comment:
this does not belong in the reversible section...its a param in every BLHeli esc

Reply:
yes, I agree.  This is actually an existing issue but in any case I've removed these lines and instead, back on the DShot page's DShot commands section I've added a "(see :ref:`SERVO_BLH_RVMASK <SERVO_BLH_RVMASK>` parameter)".  It is fairly straight forward so perhaps we don't need a length explanation.

Alternatively I could create a new short section on this BLHeli page but I'm not sure that it is actually BLHeli related.